py/examples/facepile.py

```diff
@@ -16,7 +16,7 @@ async def serve(q: Q):
     else:
         image = 'https://images.pexels.com/photos/220453/pexels-photo-220453.jpeg?auto=compress&h=750&w=1260'
         q.page['example'] = ui.form_card(box='1 1 2 2', items=[
-            ui.facepile(name='facepile', max=4, items=[
+            ui.facepile(name='facepile', value='submitted value', max=4, items=[
```

Comment:
Please add another facepile component to the example instead of munging all together so that one submits original `True` and other one submits the specified `value`. Similar to https://github.com/h2oai/wave/blob/master/py/examples/links.py.

Reply:
![2021-12-02 10 58 37](https://user-images.githubusercontent.com/15820796/144400197-0931aec9-db9d-4405-8619-7d4dc25211cb.gif)